TrafficLight/Elements.cs

```diff
@@ -5,23 +5,23 @@
 using System.Threading;
 using System.Threading.Tasks;
 
-namespace Світлофор
+namespace TrafficLight
 {
-    internal class trafficLights : color
+    internal class TrafficLight : Color
```

Comment:
Please seperet class to file. Should have one file and one class.

Reply:
Done